source/slang/slang-check-impl.h

```diff
@@ -2870,4 +2870,47 @@ namespace Slang
 
     RefPtr<EntryPoint> findAndValidateEntryPoint(
         FrontEndEntryPointRequest* entryPointReq);
+
+
+    [[maybe_unused]] static ConstructorDecl* _getDefaultCtor(StructDecl* structDecl)
```

Comment:
Why maybe_unused is needed here?

Reply:
slang-check-impl compiled to multiple compilation units, this function is only used across 2 compilation units